doc/members/accept_recovery.rst

```diff
@@ -42,28 +42,31 @@ To restore private transactions and complete the recovery procedure, members sho
 
 .. note:: The members who submit their recovery shares do not necessarily have to be the members who previously accepted the recovery.
 
-First, members should retrieve their encrypted recovery shares via the ``recovery_share`` RPC [#recovery_share]_.
+The recovery share retrieval, decryption and submission steps are conveniently performed by the ``submit_recovery_share.sh`` script as follows:
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/members/recovery_share -X GET --cacert network_cert --key member1_privk --cert member1_cert -H "content-type: application/json"
-
-Then, members should decrypt their shares using their private encryption key and the `previous` network public encryption key (output by the first node of the now-defunct service via the ``network-enc-pubk-file`` :ref:`command line option <operators/start_network:Starting the First Node>`) using `NaCl's public-key authenticated encryption <https://nacl.cr.yp.to/box.html>`_.
-
-Finally, members should submit their decrypted share to CCF via the ``recovery_share/submit`` RPC:
-
-.. code-block:: bash
-
-    $ cat submit_recovery_share.json
-    {"recovery_share": [<recovery_share_bytes>]}
-
-    $ curl https://<ccf-node-address>/members/recovery_share/submit -X POST --data-binary @submit_recovery_share.json --cacert network_cert --key member1_privk --cert member1_cert -H "content-type: application/json"
-    false
-
-    $ curl https://<ccf-node-address>/members/recovery_share/submit -X POST --data-binary @submit_recovery_share.json --cacert network_cert --key member2_privk --cert member2_cert -H "content-type: application/json"
-    true
-
-When the recovery threshold is reached, the ``recovery_share/submit`` RPC returns ``true``. At this point, the private recovery procedure is started and the private ledger is being recovered.
+    $ ./submit_recovery_share.sh --rpc-address <ccf-node-address> --member-enc-privk member0_enc_privk.pem --network-enc-pubk network_enc_pubk --cert member0_cert
+    --key member0_privk --cacert network_cert
+    HTTP/1.1 200 OK
+    content-length: 46
+    content-type: application/json
+    x-ccf-global-commit: 27
```

Comment:
Please remove the global commit header, it's deprecated, we don't want it in the documentation.